armsrc/lfops.c

```diff
@@ -451,20 +453,21 @@ static void fc(int c, int *n) {
     if(c==0) {
         dest[((*n)++)]=1;
         dest[((*n)++)]=1;
-        dest[((*n)++)]=0;
-        dest[((*n)++)]=0;
+        dest[((*n)++)]=1;
+        dest[((*n)++)]=1;
         dest[((*n)++)]=0;
         dest[((*n)++)]=0;
         dest[((*n)++)]=0;
         dest[((*n)++)]=0;
     }
+  
     //	an fc/8  encoded bit is a bit pattern of  11000000  x6 = 48 samples
```

Comment:
It looks like you've added a few bits here. The comment still states that f/8 is 11000000. Is this from when you were trying to compensate for the FPGA failure, or is this an actual improvement ? If so, the comment should be updated